WordScramble.js

```diff
@@ -1,14 +1,15 @@
 const words = ["apple", "plane", "orange", "tiger", "soccer", "basketball", "whale", "water", "lebron", "lion", "hippo", "bottle", "kyrie"];
-
+let random;
```

Comment:
Can you give this a better name? `random` doesn't tell me anything. Technically this isn't a random word, but the chosen word or the word to be guessed.